pvlib/tests/test_pvsystem.py

```diff
@@ -1689,14 +1689,37 @@ def test_PVSystem_get_irradiance():
                                        irrads['dhi'])
 
     expected = pd.DataFrame(data=np.array(
-        [[ 883.65494055,  745.86141676,  137.79352379,  126.397131  ,
-              11.39639279],
-           [   0.        ,   -0.        ,    0.        ,    0.        ,    0.        ]]),
+        [[883.65494055, 745.86141676, 137.79352379, 126.397131, 11.39639279],
+         [0., -0., 0., 0., 0.]]),
                             columns=['poa_global', 'poa_direct',
                                      'poa_diffuse', 'poa_sky_diffuse',
                                      'poa_ground_diffuse'],
                             index=times)
+    assert_frame_equal(irradiance, expected, check_less_precise=2)
+
 
+def test_PVSystem_get_irradiance_albedo():
+    system = pvsystem.PVSystem(surface_tilt=32, surface_azimuth=135)
+    times = pd.date_range(start='20160101 1200-0700',
+                          end='20160101 1800-0700', freq='6H')
+    location = Location(latitude=32, longitude=-111)
+    solar_position = location.get_solarposition(times)
+    irrads = pd.DataFrame({'dni': [900, 0], 'ghi': [600, 0], 'dhi': [100, 0],
+                           'albedo': [0.5, 0.5]},
+                          index=times)
+    # albedo as a Series
+    irradiance = system.get_irradiance(solar_position['apparent_zenith'],
+                                       solar_position['azimuth'],
+                                       irrads['dni'],
+                                       irrads['ghi'],
+                                       irrads['dhi'],
+                                       albedo=irrads['albedo'])
+    expected = pd.DataFrame(data=np.array(
+        [[895.05134334, 745.86141676, 149.18992658, 126.397131, 22.79279558],
+         [0., -0., 0., 0., 0.]]),
+        columns=['poa_global', 'poa_direct', 'poa_diffuse', 'poa_sky_diffuse',
+                 'poa_ground_diffuse'],
+        index=times)
```

Comment:
I think we need similar test for Array.

Reply:
I'm not seeing the gap to be filled by an Array test.  `PVSystem.get_irradiance` is (mostly) a wrapper for `Array.get_irradiance`, so this PVSystem test confirms that data passes through to `irradiance.get_total_irradiance` and back.

Reply:
That's true, but `Array` is an independent part of the API. So far we've been pretty good about testing that each layer of the API returns the correct values or make assertions that the correct parameters are passed through. A relatively easy approach would be to create a copy of `test_PVSystem_multi_array_get_irradiance` with values set for all of the optional method arguments.

I won't object if others think this is unnecessary.

Reply:
Makes sense. It's just that we should construct a new solution due to changing the kwargs from default. `test_PVSystem_multi_array_get_irradiance_multi_irrad` confirms that PVSystem.get_irradiance and Array.get_irradiance return consistent values.

I note a small inconsistency has crept in: Array.get_irradiance defaults to the Hay-Davies model, whereas `irradiance.get_total_irradiance` defaults to the isotropic sky diffuse model.